### PR TITLE
migrate adaptive-browser-sdk

### PR DESCRIPTION
**Pull request by Loxstomper**
Migrating https://www.npmjs.com/package/adaptive-browser-sdk to this monorepo

### CODE REVIEW

**Comment by Loxstomper** [comment]
@rramk do the existing github actions support docs/npm publishing for all of the packages contained within the `sdk` directory?

**Comment by rramk** [comment]
> @rramk do the existing github actions support docs/npm publishing for all of the packages contained within the `sdk` directory?

docs: No. Need to add a section corresponding to the specific [sdk](https://github.com/ibm-security-verify/verify-sdk-javascript/blob/main/.github/scripts/pushdocs.sh#L10). ( We could enhance to generalize to publish any sdk docs once we agree)
npm: Yes. It should push to npmjs once a git tag is pushed OR a github release is created
@Loxstomper

File: sdk/adaptive-browser-sdk/demo/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "name": "adaptive-browser-sdk-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (rramk) [inline sdk/adaptive-browser-sdk/demo/package.json:11]: Should be `MIT`

File: sdk/adaptive-browser-sdk/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "docs": "rm -r docs 2>/dev/null; npx jsdoc -c ./.jsdoc.json --verbose",
+    "codecheck": "npx eslint \"**/*.js\"",
+    "codecheckfix": "npx eslint --fix \"**/*.js\"",
+    "build": "./node_modules/browserify/bin/cmd.js lib/adaptive-v1.js -o dist/adaptive-v1.js -t [ babelify --presets [ @babel/preset-env ] ] && node_modules/.bin/babel-minify dist/adaptive-v1.js -o dist/adaptive-v1.min.js"
+  },
+  "keywords": [],
+  "author": "Adam Dorogi-Kaposi <adam.dorogi-kaposi@ibm.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/IBM-Security/adaptive-browser-sdk"
```
Comment (rramk) [inline sdk/adaptive-browser-sdk/package.json:18]: Needs to point to correct repo. Please use `directory: sdk/adaptive-browser-sdk`

**Comment by rramk** [comment]
@Loxstomper 
This is just a suggestion. Since the parent directory is `sdk`, I think it's better to change the adaptive sdk directory name to be `adaptive-browser` instead of `adaptive-browser-sdk`.

File: sdk/adaptive-browser-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Prerequisites
+- [Onboard your application](https://docs.verify.ibm.com/verify/docs/on-boarding-a-native-application).
+- Configure and set up the Proxy SDK on your server. Check out the [Adding the proxy SDK to an application](https://docs.verify.ibm.com/verify/docs/developing-a-native-web-application#add-the-proxy-sdk-to-the-application) to get started.
+
+##  Installation
+After setting up the Proxy SDK on your server, you can install the IBM Security Verify Adaptive Browser SDK by cloning this repository, or installing from [npm](https://github.com/npm/cli). Check out the [Add the web SDK](https://docs.verify.ibm.com/verify/docs/developing-a-native-web-application#add-the-proxy-sdk-to-the-application) for a detailed guide.
+
+```bash
+npm install adaptive-browser-sdk
```
Comment (rramk) [inline sdk/adaptive-browser-sdk/README.md:17]: Please change the name to `@ibm-verify/adaptive-browser-sdk`

File: sdk/adaptive-browser-sdk/lib/adaptive-v1.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-var */
+/**
+ * MIT License
+ * Copyright 2020 - IBM Corp.
```
Comment (rramk) [inline sdk/adaptive-browser-sdk/lib/adaptive-v1.js:4]: I think Craig shared a copyright statement that should be used

File: sdk/adaptive-browser-sdk/package.json (hunk 2 above)
Comment (craigaps) [reply sdk/adaptive-browser-sdk/package.json:18]: @Loxstomper the `package.json` , shouldn't repository url point to `https://github.com/IBM-Security-verify/verify-sdk-javascropt/sdk/adaptive-browser`?

Might want to change the author as well to you.

File: sdk/adaptive-browser-sdk/lib/adaptive-v1.js (hunk 4 above)
Comment (craigaps) [reply sdk/adaptive-browser-sdk/lib/adaptive-v1.js:4]:
```
//
// Copyright contributors to the IBM Security Verify {component} SDK for Javascript project
//
```

i.e `component` = `adaptive-browser`

**Review by rramk — COMMENTED** [review]
LGTM

For the docs to be published, we need the corresponding change in workflow file

**Review by rramk — APPROVED** [review]

**Pull request merged**